.vscode/cpx.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Do you think we can disable the creation of this config file if it's empty (not in this PR)?

Reply:
Yes, I'll remove this one